tests/robustness/failpoints.go

```diff
@@ -85,30 +85,59 @@ var (
 	}}
 )
 
-func triggerFailpoints(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster, config FailpointConfig) {
+func verifyClusterHealth(ctx context.Context, t *testing.T, clus *e2e.EtcdProcessCluster, failpointName string) error {
+	for i := 0; i < len(clus.Procs); i++ {
+		resp, err := clus.Client().Status(ctx)
+		if err != nil {
+			return fmt.Errorf("failed to retrieve cluster health, err: %v", err)
+		}
+		for _, memberError := range resp[i].Errors {
```

Comment:
Hey @serathius - This is the part of what I have proposed that I am the most uncertain about, can you please provide some guidance on if this is even a reasonable way to check for cluster health?  I still have much to learn here 😕

Reply:
One way to do that is check health of each cluster member by sending health request to each member.